recipe/meta.yaml

```diff
@@ -11,6 +11,7 @@ source:
     patches:
         - clang.patch  # [osx]
         - prepend-dll.patch  # [win]
+        - disable_jpeg12.patch  # [win]
```

Comment:
>  In the commit message?

Yep. That will prevent `AppVeyor` and `Travis-ci` from starting (it does not work with `CircleCI`, but we can merge it without waiting for it anyways :smile:).  If you ever need to stop only `AppVeyor` there is also `[skip appveyor]`.